app/controllers/auth0_controller.rb

```diff
@@ -20,6 +20,10 @@ def refresh_token
     render :refresh_token, layout: false
   end
 
+  def login
+    redirect_to auth0_signout_url(url_for(action: :refresh_token, only_path: false, params: { mode: "login" }))
```

Comment:
nit: line 24 is a little hard to read. Might be cleaner on two lines?

Reply:
Also maybe a comment on the mode 'login'? It seems unintuitive at first that the login action appears to go to `auth0_signout_url`

Reply:
I just figured out a better approach to this problem. I don't need to invoke the sign_out URL anymore, I need to send a parameter `prompt=login` AND `connection=...` to the authorization endpoint.
It is worth pointing out that I had tried to send `prompt=login` before without any success, but it seems that `prompt=login` only works as expected if you send the connection name as well.
I'm making changes to this PR now.